### PR TITLE
Added a blog entry for 3.6.0 release

### PR DESCRIPTION
**Pull request by satishd**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -22,6 +22,46 @@
...
+                    <a href="#apache_kafka_360_release_announcement">Apache Kafka 3.6.0 Release Announcement</a>
+                </h2>
+                15 Sep 2023 - Satish Duggana (<a href="https://twitter.com/0xeed">@SatishDuggana</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.6.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.6.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_6_0">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>The ability to migrate Kafka clusters from ZK to KRaft mode with no downtime is still an early access feature. It is currently only suitable for testing in non production environments. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-866+ZooKeeper+to+KRaft+Migration">KIP-866</a> for more details.</p>
```
Comment (showuon) [inline blog.html:33]: This might need @mumrah 's confirmation.

File: blog.html [hunk 2]
```diff
@@ -22,6 +22,46 @@
...
+                <p><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage">Tiered Storage<a> is still an early access feature. It is currently only suitable for testing in non production environments. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Tiered+Storage+Early+Access+Release+Notes">Early Access Notes</a> for more details.</p>
```
Comment (showuon) [inline blog.html]: <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage">Tiered Storage<a> is [no still] an early access feature....

File: blog.html [hunk 3]
```diff
@@ -22,6 +22,46 @@
...
+
+                <p><it>Note: ZooKeeper is now marked deprecated in this release. ZooKeeper is planned to be removed in Apache Kafka 4.0. (Cf <a href="/documentation#zk_depr">ZooKeeper Deprecation</a>)</it><p>
```
Comment (showuon) [inline blog.html]: ZK is marked as depracated in v3.5.0, not v3.6.0. So, maybe:
`Zookeeper is marked as deprecated. Zookeeper is planned to be removed in Apache Kafka 4.0. `

**Review by showuon — COMMENTED** [review]
Left some comments. Thanks for the PR.

File: blog.html (hunk 3 above)
Comment (satishd) [reply blog.html]: Changed it to `ZooKeeper is marked as deprecated since the 3.5.0 release.`

**Comment by satishd** [comment]
Thanks @showuon for the review. Addressed them with the latest commit.

**Review by showuon — COMMENTED** [review]
Let's get this PR merged to unblock the RC build.
@mumrah can update the ZK migrating to KRaft part later.

**Review by showuon — APPROVED** [review]

**Comment by satishd** [comment]
@mumrah I am merging this PR that added a blog for 3.6.0 for now. We can have followup PRs if needed.

**Comment by mumrah** [comment]
@satishd @showuon I think we need to revert this change until the 3.6 release is complete. Here's what I just sent out on the mailing list RC0 thread:

> I noticed that the PR for the 3.6 blog post is merged. This makes the blog post live on the Kafka website https://kafka.apache.org/blog.html. The blog post (along with other public announcements) is usually the last thing we do as part of the release. I think we should probably take this down until we're done with the release, otherwise users stumbling on this post could get confused. It also contains some broken links.

**Comment by jlprat** [comment]
I just reverted it and merged it. @satishd feel free to "reopen" this one again.

**Comment by satishd** [comment]
Thanks @mumrah for clarifying the merge timeline for the blog section. I will open a PR and wait for the release candidate to be finalized before merging it. 
I will also update the wiki to make it clear that the blog should be merged only after the release candidate is finalized and it is the last thing that should be done as part of the release.

File: blog.html (hunk 1 above)
Comment (todiaz) [inline blog.html:33]: non production -> non-production

**Pull request merged**